### PR TITLE
Run tests in the container

### PR DESCRIPTION
**Pull request by rpitonak**
Tests now running inside the container, but still failing because of 'baseruntime' and 'microdnf' tests (Issue [#45](https://github.com/fedora-modularity/meta-test-family/issues/45))

Also failing because of behave tests, @jscotka can you take a look on it?
```
make[3]: Leaving directory '/src/tests'
cd ../../behave-tests; VERSION=x.y MODULE=docker URL="docker=image:x.y" make all
make[3]: Entering directory '/src/behave-tests'
behave
make[3]: behave: Command not found
```

### CODE REVIEW

**Comment by TomasTomecek** [comment]
you may need to install `python2-behave`

**Comment by TomasTomecek** [comment]
I propose to remove behave tests from this repository. Once that's done, I think the test suite should be passing just fine.

We should also polish tests in `tests/`, I'll open an issue for that.

File: tests/Makefile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-CMD=avocado run
+CMD=avocado run --show-job-log
```
Comment (TomasTomecek) [inline tests/Makefile]: I guess let's remove the `--show-job-log` and just mention it in comment

**Review by TomasTomecek — CHANGES_REQUESTED** [review]
looks good! tests did pass.

follow-up work: https://github.com/container-images/container-image-template/issues/39

**Comment by TomasTomecek** [comment]
In this case, I consider https://github.com/fedora-modularity/meta-test-family/issues/64 to be a blocker, since we are not able to figure out why the tests failed.

File: x.y/centos/Makefile [hunk 2]
```diff
@@ -23,4 +23,3 @@ run: build
 
 test: build
 	cd ../../tests; VERSION=$(VERSION) DOCKERFILE="../centos/Dockerfile" MODULE=docker URL="docker=$(IMAGE_NAME):$(VERSION)" make all
```
Comment (phracek) [inline x.y/centos/Makefile:25]: This is wrong reference to Dockerfile. It should be `../x.y/centos/Dockerfile`

File: x.y/fedora/Makefile [hunk 3]
```diff
@@ -23,4 +23,3 @@ run: build
 
 test: build
 	cd ../../tests; VERSION=$(VERSION) DOCKERFILE="../fedora/Dockerfile" MODULE=docker URL="docker=$(IMAGE_NAME):$(VERSION)" make all
```
Comment (phracek) [inline x.y/fedora/Makefile:25]: This is wrong reference to Dockerfile. It should be `../x.y/fedora/Dockerfile`

File: x.y/rhel/Makefile [hunk 4]
```diff
@@ -23,4 +23,3 @@ run: build
 
 test: build
 	cd ../../tests; VERSION=$(VERSION) DOCKERFILE="../rhel/Dockerfile" MODULE=docker URL="docker=$(IMAGE_NAME):$(VERSION)" make all
```
Comment (phracek) [inline x.y/rhel/Makefile:25]: This is wrong reference to Dockerfile. It should be `../x.y/rhel/Dockerfile`

File: x.z/centos/Makefile [hunk 5]
```diff
@@ -23,4 +23,3 @@ run: build
 
 test: build
 	cd ../../tests; VERSION=$(VERSION) DOCKERFILE="../centos/Dockerfile" MODULE=docker URL="docker=$(IMAGE_NAME):$(VERSION)" make all
```
Comment (phracek) [inline x.z/centos/Makefile:25]: This is wrong reference to Dockerfile. It should be `../x.y/centos/Dockerfile`

File: x.z/fedora/Makefile [hunk 6]
```diff
@@ -23,4 +23,3 @@ run: build
 
 test: build
 	cd ../../tests; VERSION=$(VERSION) DOCKERFILE="../fedora/Dockerfile" MODULE=docker URL="docker=$(IMAGE_NAME):$(VERSION)" make all
```
Comment (phracek) [inline x.z/fedora/Makefile:25]: This is wrong reference to Dockerfile. It should be `../x.z/fedora/Dockerfile`

**Comment by phracek** [comment]
I will file a new PR based on this.

**Comment by TomasTomecek** [comment]
addressed in the other pr

**Pull request closed without merging**